src/mono/mono.proj

```diff
@@ -352,13 +352,16 @@
       <_MonoRunInitCompiler>false</_MonoRunInitCompiler>
     </PropertyGroup>
     <ItemGroup Condition="'$(TargetsAndroid)' == 'true'">
-      <_MonoCMakeArgs Include="-DCMAKE_SYSTEM_NAME=Android"/>
```

Comment:
I think we still need `CMAKE_SYSTEM_NAME` - it's used in src/mono/CMakeLists.txt'

Reply:
It's set by the NDK toolchain itself